app/uk/gov/hmrc/eoricommoncomponent/frontend/views/helpers/paragraph.scala.html

```diff
@@ -15,7 +15,7 @@
  *@
 
 @this()
-@(id: Option[String] = None, text: String = "", html: Option[Html] = None)
+@(text: String = "", id: Option[String] = None, html: Option[Html] = None)
```

Comment:
Swapped around order of parameters. We don't always require an ID but we will always require content.